src/1-foundations/1-type-theory/exercises/1.11-solution.rzk.md

```diff
@@ -8,24 +8,31 @@ This is a literate Rzk file:
 
 ## Problem statement
 
-!!! warning
+!!! question "Exercise 1.11"
+    Show that for any type $A$, we have $¬¬¬A → ¬A$.
 
-    To be done.
+## Solution
 
-## Solution (part 1)
+This is a weakened version of the double negation law and it holds constructively.
 
-!!! warning
+Proposition $¬X$ corresponds to `(X → ⊥)` type.
+We are given `f : (((A → ⊥) → ⊥) → ⊥)` and `a : A ` and we need to construct an element of type `⊥`.
+For that, we need to apply `f` to a term of type `((A → ⊥) → ⊥)`. 
+The term `\ g → g a` has the required type.
 
-    To be done.
-
-## Solution (part 2)
-
-!!! warning
-
-    To be done.
-
-## Solution (part ...)
+```rzk
+#def triple-neg
+  (A : U)
+  : (((A → ⊥) → ⊥) → ⊥) → (A → ⊥)
```

Comment:
Techincally, `⊥` here is not the empty type, but it will be with the planned proper inclusion of shapes into types.

Reply:
It typechecks anyway, so I do not mind :)